src/Analysis/Ast/Impl/Types/PythonType.cs

```diff
@@ -59,7 +59,10 @@ internal class PythonType : LocatedMember, IPythonType {//, IEquatable<IPythonTy
         public virtual string Documentation { get; private set; }
         public virtual BuiltinTypeId TypeId => _typeId;
         public bool IsBuiltin => DeclaringModule == null || DeclaringModule is IBuiltinsPythonModule;
-        public virtual bool IsAbstract => false;
+        public virtual bool IsAbstract {
```

Comment:
Should not allow setters on public interface

Reply:
Can you give me your reasoning for this?

Reply:
So users of the analysis, such as extensions like IntelliCode would not be able to modify completed analysis.